tests/robustness/failpoint/network.go

```diff
@@ -57,6 +57,11 @@ func (tb triggerBlackhole) Trigger(ctx context.Context, t *testing.T, member e2e
 }
 
 func (tb triggerBlackhole) Available(config e2e.EtcdProcessClusterConfig, process e2e.EtcdProcess) bool {
+	// Avoid triggering failpoint if waiting for failpoint would take too long to fit into timeout.
+	// Number of required entries for snapshot depends on etcd configuration.
+	if tb.waitTillSnapshot && entriesToGuaranteeSnapshot(config) > 200 {
```

Comment:
If we want to completely avoid this, should this be 200?
I see we are randomly picking `SnapshotCount` from 50, 100 and 1000. If 50n is picked and `SnapshotCatchupEntries` is 100 then this branch will not be taken if I understand correctly.

Reply:
Nvm, I think I understand now. 50 is within the timeout.

Reply:
Possible steps of `SnaphostCount` + `SnapshotCatchupEntries`:
* `150`
* `200`
* `1100`
* `5050`
* `5100`
* `6000`

`200` takes around 10 seconds on my machine which is ok.

The next jump would be to `1100` which is too big.